lib/ember-cli/deploy/redis.rb

```diff
@@ -0,0 +1,82 @@
+require "active_support/core_ext/object/blank"
+require "ember-cli/deploy/page"
+require "redis"
+
+module EmberCLI
+  module Deploy
+    class Redis
+      def initialize(namespace:, index_html: nil)
```

Comment:
This is very interesting.

We're running into a similar issue on my project, we'll have to give this a try.